scanpipe/templates/scanpipe/base.html

```diff
@@ -5,6 +5,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{% block title %}ScanCode.io{% endblock %}</title>
+    <link rel="icon" href="{% static 'favicon.ico' %}" type="image/x-icon"/>
```

Comment:
This is unrelated to this PR.

Reply:
You can merge main into your branch once in a while to get the latest changes such as this one that is now in main ;)

Reply:
Thanks;) 
Got a merge conflict :)

Reply:
This conflict should be trivial to fix.